frontend/public/components/search.tsx

```diff
@@ -110,6 +122,50 @@ const SearchPage_: React.FC<SearchProps> = (props) => {
     clearLabelFilter();
   };
 
+  const unPin = (resource: string) => {
+    const label = modelFor(resource)?.labelPlural;
+    const title = (
+      <>
+        <YellowExclamationTriangleIcon className="co-icon-space-r" /> Unpin from navigation?
+      </>
+    );
+    const message = (
+      <span>
+        Are you sure you want to unpin <strong>{label}</strong> from navigation?
+      </span>
+    );
+    confirmModal({
```

Comment:
This modal is duplicate code.

I don't agree that we need a modal popup from the search page. But that's a UX decision.

Reply:
Added it there because accidental unpin'ing causes you to lose the order of the navigation pinned items. If a mistake is made you'd have to unpin/pin a bunch of items (requiring you to re-select them in the search page) to get that order back.